packages/block-library/src/navigation/index.php

```diff
@@ -166,121 +144,25 @@ function render_block_core_navigation( $content, $block ) {
 		isset( $attributes['className'] ) ? array( $attributes['className'] ) : array(),
 		( isset( $attributes['orientation'] ) && 'vertical' === $attributes['orientation'] ) ? array( 'is-vertical' ) : array(),
 		isset( $attributes['itemsJustification'] ) ? array( 'items-justified-' . $attributes['itemsJustification'] ) : array(),
-		isset( $attributes['align'] ) ? array( 'align' . $attributes['align'] ) : array()
+		isset( $attributes['align'] ) ? array( 'align' . $attributes['align'] ) : array(),
+		isset( $attributes['showSubmenuIcon'] ) && $attributes['showSubmenuIcon'] ? array( 'has-submenu-icon' ) : array()
```

Comment:
Where is `.has-submenu-icon` used?

Reply:
My solution before block context existed was to apply a classname to the parent block to hide the submenu icon.

When rebasing I completely got rid of my scss changes though, so it looks like this was left behind. I think it can be removed.

Reply:
c44169b7ba729a7c6bfdd6695cca7b6ef1d86046